samples/subsys/llext/shell_loader/README.rst

```diff
@@ -0,0 +1,71 @@
+.. zephyr:code-sample:: llext-shell-loader
+   :name: Linkable loadable extensions shell module
+   :relevant-api: llext
+
+   Manage loadable extensions using shell commands.
+
+Overview
+********
+
+This example provides shell access to the :ref:`llext` system and provides the
+ability to manage loadable code extensions in the shell.
+
+Requirements
+************
+
+A board with shell capable console.
+
+Building
+********
+
+.. zephyr-app-commands::
+   :zephyr-app: samples/subsys/llext/shell_loader
+   :goals: build
+   :compact:
+
+*NOTE* you may need to disable memory for the sample to work (e.g. CONFIG_ARM_MPU=n)
+
+Running
+*******
+
+Once the board has booted, you will be presented with a shell prompt.
+All the llext system related commands are available as sub-commands of llext
+which can be seen with llext help
+
+.. code-block:: console
+
+  uart:~$ llext help
+  llext - Loadable extension commands
+  Subcommands:
+    list          :List loaded extensions and their size in memory
+    load_hex      :Load an elf file encoded in hex directly from the shell input.
+                   Syntax:
+                   <ext_name> <ext_hex_string>
+    unload        :Unload an extension by name. Syntax:
+                   <ext_name>
+    list_symbols  :List extension symbols. Syntax:
+                   <ext_name>
+    call_fn       :Call extension function with prototype void fn(void). Syntax:
+                   <ext_name> <function_name>
+
+On arm boards a hello world elf can be found in tests/subsys/llext/src/hello_world_armv7_thumb.elf
```

Comment:
There was an elf in the commits but its since been dropped as binaries are bad and all that.

It's kind of unfortunate that I can't supply the elf directly here. I can leave the hex dump of the hello world for arm in the docs as is done already noting how it was created with arm gcc and xxd. Would that suffice you think?

Reply:
Ya I think that would work. It might be challenging to make sure the hexdump doesn't get stale, but as long as instructions on how to use gcc and xxd to generate it are there, I think that's all good (in fact probably even better than when the .elf file just happened to "magically" be there without using the opportunity to (re-)explain how one may build it.
Thanks!